CHANGELOG_PENDING.md

```diff
@@ -15,6 +15,7 @@
 #### Transcoder
 
 ### Bug Fixes 🐞
+- [task-runner \#24](https://github.com/livepeer/task-runner/issues/24) rendition resolution fix for portrait input videos; Min resolution applied for Nvidia hardware (@AlexKordic)
```

Comment:
Should this link to the PR rather than / as well as the issue it fixes?

Reply:
I am not sure, being another project. What do you think how to phrase this item?

Reply:
Fixed to look normal